tests/emails.test.js

```diff
@@ -0,0 +1,104 @@
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const mongoose = require('mongoose');
+const {server} = require('../src/app');
+chai.should();
+chai.use(chaiHttp);
+
+describe("POST Tests Emails",()=>{
+    it("POST /emails Not Found", (done) =>{
```

Comment:
Codacy has a fix for the issue: [Missing space after =>.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
    it("POST /emails Not Found", (done) => {
```